app/src/main/java/com/fstyle/androidtrainning/screen/ViewPagerAdapter.java

```diff
@@ -0,0 +1,43 @@
+package com.fstyle.androidtrainning.screen;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import com.fstyle.androidtrainning.screen.tabfavorite.TabFavoriteFragment;
+import com.fstyle.androidtrainning.screen.tabhome.TabHomeFragment;
+import com.fstyle.androidtrainning.screen.tabprofile.TabProfileFragment;
+import com.fstyle.androidtrainning.screen.tabsearch.TabSearchFragment;
+
+/**
+ * Created by huynh on 12/12/2017.
+ */
+
+public class ViewPagerAdapter extends FragmentPagerAdapter {
+    private static int TOTAL_TAB = 4;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName